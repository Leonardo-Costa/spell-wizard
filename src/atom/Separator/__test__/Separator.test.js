import React from 'react';
import renderer from 'react-test-renderer';

import {Separator} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<Separator />).toJSON();
    expect(tree).toMatchSnapshot();
  });