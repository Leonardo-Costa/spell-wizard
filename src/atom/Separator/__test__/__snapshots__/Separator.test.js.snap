// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#202020",
        "height": 1.5,
        "marginVertical": 20,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
